include/cute/util/sycl_vec.hpp

```diff
@@ -1,17 +1,53 @@
+/***************************************************************************************************
+ * Copyright (c) 2023 - 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Thank you for addressing my previous comments. I think you missed this change in the license here.
```suggestion
 * Copyright (c) 2024 - 2024 Codeplay Software Ltd. All rights reserved.
```